test/accum.c

```diff
@@ -2194,7 +2194,7 @@ test_swmr_write_big(bool newest_format)
         ZeroMemory(&pi, sizeof(pi));
 
         if (0 == CreateProcess(NULL, SWMR_READER, NULL, NULL, false, 0, NULL, NULL, &si, &pi)) {
-            printf("CreateProcess failed (%d).\n", GetLastError());
+            printf("CreateProcess failed (%lu).\n", GetLastError());
```

Comment:
This should be PRIu32 so it's always true

https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-dtyp/262627d8-3418-4627-9218-4ffe110850b2

Reply:
I tried but it doesn't work:
```
cmake --build . --config Debug
[1/2] Building C object test\CMakeFiles\accum.dir\accum.c.obj
C:\Users\JoeLee\source\repos\hdf5\test\accum.c(2197,61): warning: format specifies type 'unsigned int' but the argument has type 'DWORD' (aka 'unsigned long') [-Wformat]
            printf("CreateProcess failed (%" PRIu32 ").\n", GetLastError());
                                          ~~~~~~~~~         ^~~~~~~~~~~~~~
1 warning generated.
[2/2] Linking C executable bin\accum.exe
```